Services/Client.php

```diff
@@ -0,0 +1,48 @@
+<?php
+namespace Nietonfir\RaygunBundle\Services;
+
+use Raygun4php\RaygunClient;
+
+class Client extends RaygunClient implements NietonfirRaygunClient
```

Comment:
This is a very Symfony standard in naming. Plus, the full namespace is  `Nietonfir\RaygunBundle\Services`. I'd rather change `Client` into something more meaningful, like `NietonfirRaygunClient`. What do you think?